tests/testthat/test-post.R

```diff
@@ -1,12 +1,42 @@
 test_that("upload_media_to_twitter() can handle small file", {
   skip_on_cran()
-  
-  id <- upload_media_to_twitter(test_path("tweet.gif"))
+
+  # gif
+  path <- test_path("tweet.gif")
+  fs <- file.size(path)
+
+  ## test-unchunked
+  id <- upload_media_to_twitter(path, chunk_size = fs + 3)
   expect_type(id, "character")
   # doesn't appear to be any way to get info about a media object
   # after it's been uploaded
-  
-  id <- upload_media_to_twitter(test_path("tweet.gif"), chunk_size = 1024)
+
+  ## test-chunked
+  id <- upload_media_to_twitter(path, chunk_size = fs - 3)
+  expect_type(id, "character")
+
+  # png
+  path <- test_path("tweet.png")
```

Comment:
Unless you think there's something materially different in the treatment of pngs and gifs, I think you should omit this test as it doesn't actually touch any more lines of code in this package.

Reply:
Ok, I am leaving just gif and removing also `tweet.png` file